docusaurus/docs/dev-docs/api/plugins/admin-panel-api.md

```diff
@@ -367,12 +295,9 @@ export default {
 
 ### Injection Zones API
 
-Injection zones refer to areas of a view's layout where a plugin allows another to inject a custom React component (e.g. a UI element like a button).
-
-Plugins can use:
+Injection zones are areas where plugins can inject a custom React component.
 
-* Strapi's [predefined injection zones](#using-predefined-injection-zones) for the Content Manager,
-* or custom injection zones, created by a plugin
+Plugins can either use Strapi's [predefined injection zones](#using-predefined-injection-zones) or create custom zones.
```

Comment:
I would keep the previous wording, or at least bullet points. Using bullet points when enumerating is a best practice in technical writing.